### ISSUE TITLE
About Pretraining weight in 2.2%, 5%

### ISSUE THREAD

**Issue by JooYJang** [licongguan/ILM-ASSL#3]
Hi, I'm interested with Self training + Active Learning Concept and want to reproduce the results as paper suggested

However, as I saw the log of 2.2%, 5% that you sent, I am confused about the iterative loop.

As far as I understand, 2.2% and 5% is 2nd iteration and the pretrained weight should be from 1% model's final output.
However, the log is telling that pretrained model is from Imagenet.

If I'm wrong, please let me know.

Sincerely, Joo Young Jang

Following result is 5% starting log.

set random seed to 1
[Info] Load ImageNet pretrain from '/media/dell/Elements/DATA/core/models/resnet101.pth' 
missing_keys:  [] 
unexpected_keys:  ['fc.weight', 'fc.bias']
[Info] Load ImageNet pretrain from '/media/dell/Elements/DATA/core/models/resnet101.pth' 
missing_keys:  [] 
unexpected_keys:  ['fc.weight', 'fc.bias']
[Info] Load ImageNet pretrain from '/media/dell/Elements/DATA/core/models/resnet101.pth' 
missing_keys:  [] 
unexpected_keys:  ['fc.weight', 'fc.bias']
[Info] Load ImageNet pretrain from '/media/dell/Elements/DATA/core/models/resnet101.pth' 
missing_keys:  [] 
unexpected_keys:  ['fc.weight', 'fc.bias']
[2022-10-25 20:21:07,206][        base.py][line:  41][    INFO] # samples: 2150
[2022-10-25 20:21:07,208][        base.py][line:  41][    INFO] # samples: 2150
labeled:  4825
labeled:  4825
[2022-10-25 20:21:07,223][        base.py][line:  41][    INFO] # samples: 2825
[2022-10-25 20:21:07,226][        base.py][line:  41][    INFO] # samples: 2825
unlabeled:  2825
unlabeled:  2825
[2022-10-25 20:21:07,242][        base.py][line:  41][    INFO] # samples: 500
[2022-10-25 20:21:07,242][        base.py][line:  41][    INFO] # samples: 500
[2022-10-25 20:21:07,242][     builder.py][line:  28][    INFO] Get loader Done...
[2022-10-25 20:21:07,242][     builder.py][line:  28][    INFO] Get loader Done...
No checkpoint found in 'checkpoints/ckpt.pth'
[2022-10-25 20:21:07,255][   lr_helper.py][line:  65][    INFO] The kwargs for lr scheduler: 0.9
[2022-10-25 20:21:07,257][   lr_helper.py][line:  65][    INFO] The kwargs for lr scheduler: 0.9
epoch [ 0 : ] sample_rate_target_class_conf [0.10357965 0.0711445  0.04550922 0.09053792 0.05731867 0.11211205
 0.07049027 0.05302454 0.08098789 0.0774475  0.04990353 0.0625474
 0.04623231 0.07916455]
epoch [ 0 : ] criterion_per_class tensor([0.0901, 0.8649, 0.4227, 0.1821, 0.7204, 0.9933, 0.4074, 0.9022, 0.9825,
        0.0326, 0.3988, 0.7301, 0.9449, 0.5666, 0.6234, 0.9697, 0.8400, 0.9904,
        0.5960], device='cuda:0')
epoch [ 0 : ] sample_rate_per_class_conf tensor([0.9406, 0.1396, 0.5967, 0.8455, 0.2890, 0.0069, 0.6126, 0.1011, 0.0181,
        1.0000, 0.6215, 0.2790, 0.0570, 0.4481, 0.3893, 0.0314, 0.1654, 0.0099,
        0.4176], device='cuda:0')

**Comment by licongguan** [comment]
The purpose of our first training with semi-supervised (self-training) is to obtain a warm-up model that can help active learning to make better choices. In the subsequent iterative process, we still use the same pre-training, but replace the labeled samples. But your idea is good, you can try to use 1% model in the second iteration, if the effect is better, you can pull request.

**Comment by JooYJang** [comment]
Okay Thanks!
